Library/Homebrew/diagnostic.rb

```diff
@@ -345,6 +338,40 @@ def check_access_prefix_directories
         EOS
       end
 
+      def check_missing_prefix_directories
+        return if HOMEBREW_PREFIX.to_s != "/usr/local"
+        missing_dirs = Keg::ALL_TOP_LEVEL_PATHS.reject(&:exist?)
+        return if missing_dirs.empty?
+
+        <<~EOS
+          The following directories are missing:
+          #{missing_dirs.join("\n")}
+          You should create them:
```

Comment:
`create and change ownership of them:` and then can kill `And change the permissions of these directories:` below